src/Native/Bootstrap/main.cpp

```diff
@@ -447,6 +448,7 @@ int main(int argc, char* argv[])
         retval = -1;
     }
 #endif
+    RhpManagedShutdown();
```

Comment:
I would be better to do this in the `StartupCodeHelpers.Shutdown` that is called right after Main method returns.

Reply:
Apparently this fixed issue with `GC_Scenarios._BinTree_XXX` tests.